web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java

```diff
@@ -0,0 +1,16 @@
+package nl.tehdreamteam.se42.web.soap.user;
+
+import nl.tehdreamteam.se42.web.controller.UserController;
+
+import javax.jws.WebParam;
+import javax.jws.WebService;
+
+@WebService
+public class SoapUserController implements UserController {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document this public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)